docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,414 @@
 ```
+=====================================
+| serve.start_year | serve.end_year |
+=====================================
+| 1997             | 2016           |
+-------------------------------------
+```
+
+### Updating Data
+
+You can update the vertices and edges you just inserted.
+
+#### Updating Vertices
+
+The following example shows you how to change the _age_ value of vertex `100` from `42` to `41`.
 
-### Update Data
+Enter the following statement to update the _age_ value:
 
 ```ngql
--- Update vertex
-nebula> UPDATE VERTEX 101 SET course.credits = $^.course.credits + 1, building.name = "No8" YIELD $^.course.name AS Name, $^.course.credits AS Credits, $^.building.name
--- Update edge
-nebula> UPDATE EDGE 200 -> 101 OF choose SET grade = choose.grade +1
+nebula> UPDATE VERTEX 100 SET player.age = $^.player.age - 1
+```
+
+**Note**: `$^` represents the source vertex.
+
+To check whether the _age_ value is updated, enter the following statement:
+
+```ngql
+nebula> FETCH PROP ON player 100;
+```
+
+The following information is displayed:
+
+```
+============================
+| player.name | player.age |
+============================
+| Time        | 41         |
+----------------------------
 ```
 
-### Fetch Data
+#### Updating Edges
 
-Fetch the data updated.
+The following example shows you how to change the value of the _degree_ attribute in the _follow_ edge between vertex `100` and vertex `101`. Now we change the _degree_ attribute from `95` to `96`.
+
+Enter the following statement to update the _degree_ value:
+
+```ngql
+nebula> UPDATE EDGE 100-> 101 OF follow SET degree = follow.degree +1
+```
+
+To check whether the _degree_ value is updated, enter the following statement:
 
 ```ngql
-nebula> FETCH PROP ON course 101;
-================================
-| course.name | course.credits |
-================================
-| Math        | 4              |
---------------------------------
-nebula> FETCH PROP ON choose 200 -> 101
-================
-| choose.grade |
-================
-| 6            |
-----------------
+nebula> FETCH PROP ON follow 100 -> 101;
 ```
 
-### Delete Data
+The following information is displayed:
+
+```
+=================
+| follow.degree |
+=================
+| 96            |
+-----------------
+```
+
+### Deleting  Data
+
+If you have some data that you do not need, you can delete it from you graph database.
 
-Delete vertex.
+To delete a vertex whose ID is `121`, enter the following statement:
 
 ```ngql
-nebula> DELETE VERTEX $vid
+nebula> DELETE VERTEX 121;
 ```
 
+To check whether the vertex is deleted, enter the following statement;
+
+```ngql
+nebula> FETCH PROP ON player 121;
+```
+
+The following information is displayed:
+
+```
+Execution succeeded (Time spent: 1571/1910 us)
+```
+
+**Note**: The above return information indicates the query operation is successful but no data is queried from your graph database because the data is deleted.
+
 ### Sample Queries
```

Comment:
a simple explanation of 

Delete Edge

DELETE EDGE <edge_type> <vid> -> <vid>

no example is ok